.github/stale.yml

```diff
@@ -0,0 +1,45 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Issues or Pull Requests with these labels will never be
+# considered stale. Set to `[]` to disable
+---
+exemptLabels:
+  - keepalive
+  - security
+  - reliability
+  - good first issue
```

Comment:
these labels need to be created

Reply:
Yeah, this plus integration needs to be done

Reply:
We can create these after review is finished

Reply:
Sounds good. Thank you

Reply:
I'm more in favor of mapping the labels we have / need rather than creating labels for stuff speculatively.

Furthermore, based on how we've done things in go-ceph so far I'd propose the following behavior:
* Unlabeled issues go stale
* Issues labeled with "question" go stale
* No other issues go stale automatically

I'm on the fence WRT making PRs go stale at all. If you think PRs should go stale please make your case.

Reply:
> Unlabeled issues go stale

Couldn't find a way where it makes only unlabeled issues stale because the bot works on labels.
Config has two options:
- onlyLabel: specify the label you want to check
- exemptLabel: specify the label you want to exempt
Best option is to label the issue with question and configure the bot to check only such issues for stale.


> Issues labeled with "question" go stale

This can be done, bot can work only on specified labels

> No other issues go stale automatically

If you set it to work on specified label no issue with any other label would be checked.

> I'm on the fence WRT making PRs go stale at all. If you think PRs should go stale please make your case.

That's fine, we can set the bot only for issues

Reply:
> > Unlabeled issues go stale
> 
> Couldn't find a way where it makes only unlabeled issues stale because the bot works on labels.
> Config has two options:
> 
>     * onlyLabel: specify the label you want to check
> 
>     * exemptLabel: specify the label you want to exempt
>       Best option is to label the issue with "question" and configure the bot to check only such issues for stale.
> 

That's unfortunate. But I suppose we can live with those for now. I do wonder if they have an issue tracker where we can request it as a feature.

> 
> > Issues labeled with "question" go stale
> 
> This can be done, bot can work only on specified labels
> 
> > No other issues go stale automatically
> 
> If you set it to work on specified label, issues with any other label would not be checked.
> 
> > I'm on the fence WRT making PRs go stale at all. If you think PRs should go stale please make your case.
> 
> That's fine, we can set the bot only for issues

Yeah, that all sounds good to me. We only have one or two "stale" PRs, nothing we can't handle manually for now. So lets start off with just aging on "question" labels only and we can expand on that later if needed. We've been fairly diligent about adding labels so far.